system-tests/lib/cre/capabilities/node_config.go

```diff
@@ -14,17 +15,22 @@ import (
 func MakeBinariesExecutable(customBinariesPaths map[cre.CapabilityFlag]string) error {
 	for capabilityFlag, binaryPath := range customBinariesPaths {
 		if binaryPath == "" {
-			return fmt.Errorf("binary path for capability %s is empty", capabilityFlag)
+			return fmt.Errorf("binary path for capability %s is empty. Please set the binary path in the capabilities TOML config", capabilityFlag)
 		}
 
 		// Check if file exists
 		if _, err := os.Stat(binaryPath); os.IsNotExist(err) {
-			return fmt.Errorf("binary path %s for capability %s does not exist", binaryPath, capabilityFlag)
+			absPath, absErr := filepath.Abs(binaryPath)
+			if absErr != nil {
+				return errors.Wrapf(absErr, "failed to get absolute path for binary %s", binaryPath)
```

Comment:
Can we stick to the std lib? We've been trying to migrate away from `pkg/errors` for a long time now
```suggestion
				return fmt.Errorf(absErr, "failed to get absolute path for binary %s: %w", binaryPath, absErr)
```

Reply:
I'd rather not, because I find that stack trace to be very useful.

Reply:
Do you actually need whole traces? or just file/line information? https://github.com/jmank88/callerr